pkg/cmd/admin/registry/registry.go

```diff
@@ -224,6 +231,23 @@ func (opts *RegistryOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command,
 		opts.nodeSelector = valid
 	}
 
+	if len(opts.Config.FSGroup) > 0 {
+		if val, err := strconv.ParseInt(opts.Config.FSGroup, 10, 64); err != nil || val == 0 {
```

Comment:
Looks like this allows the values less than zero. Perhaps `uint32` is more suitable in this case.